apis/r/tests/testthat/test-query-condition.R

```diff
@@ -15,6 +15,8 @@ test_that("DataFrame Factory", {
       arrow::field("uint32", arrow::uint32()),
       arrow::field("uint64", arrow::uint64()),
       arrow::field("string", arrow::string()),
+      # Unlike in pyarrow there is no arrow::large_string
+      arrow::field("utf8", arrow::utf8()),
```

Comment:
There is `arrow::large_utf8()` in Arrow-R

Reply:
Indeed, and I test `large_utf8` on this PR -- thank you!